shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -557,20 +557,6 @@ + (NSString*)generateThreadLabel:(NSString*)labelPrefix {
           threadHostType};
 }
 
-static void SetEntryPoint(flutter::Settings* settings, NSString* entrypoint, NSString* libraryURI) {
```

Comment:
nit: Inlining this function is a slight step backwards.  Anywhere where we can make meaningful abstractions with functions is a win, especially when using functions which reduce the scope of possible side effects =)

Reply:
Sure, it was a random drive-by. Reverting